_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -420,3 +420,24 @@ function jetpack_current_user_data() {
 
 	return $current_user_data;
 }
+
+/**
+ * Set the admin language, based on user language.
+ *
+ * @since 4.5.0
+ *
+ * @return string
+ *
+ * @todo Remove this function when WordPress 4.8 is released
+ * and replace `jetpack_get_locale()` in this file with `get_user_locale()`.
+ */
+function jetpack_get_locale() {
+	global $wp_version;
+	$locale = get_locale();
+
+	if ( version_compare( $wp_version , '4.6.1', '>') ) {
```

Comment:
Instead of using `version_compare`, how about checking if the `get_user_locale` function exists?